models.py

```diff
@@ -53,8 +54,11 @@ def from_row(cls, row: Row) -> "PayLink":
             data["max"] /= data["fiat_base_multiplier"]
         return cls(**data)
 
-    def lnurl(self, req: Request) -> str:
+    def lnurl(self, req: Request) -> str:        
         url = req.url_for("lnurlp.api_lnurl_response", link_id=self.id)
+        # Check if url is .onion and change to http
+        if ".onion" in url.hostname:
```

Comment:
- this rule is valid for `https://eat.some.onion.lnbits.com`
- should use `ends_with`